DFC.App.MatchSkills/Views/OccupationSearch/Body.cshtml

```diff
@@ -6,7 +6,7 @@
 
         <div class="govuk-grid-column-two-thirds">
 
-            <form action="GetOccupationSkills"  method="post">
+            <form action="/matchskills/body/selectskills"  method="post">
```

Comment:
you can remove the Matchskills from here/ /bod/{endpoint} will work. 
Also can we use the PageId.SelectSkills const please?

Reply:
fixed